src/outputs/harbor_output.ml

```diff
@@ -504,9 +504,10 @@ module Make (T : T) = struct
                    on_connect ~protocol ~uri ~headers:h_headers ip )) )
 
       method send b =
-        let slen = String.length b in
+        let slen = Strings.length b in
+        failwith "TODO"; let b = assert false in
```

Comment:
@toots I need some help for this last one, I am not really sure about the semantics of everything (burst in particular)... I guess that some `Buffer.t` can be replaced by `String.t`.